portfolio/pom.xml

```diff
@@ -23,21 +23,30 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+   </dependency>
+
+
+
+   
+
+
   </dependencies>
 
   <build>
     <plugins>
-      <!-- Provides `mvn package appengine:run` for local testing
-           and `mvn package appengine:deploy` for deploying. -->
       <plugin>
-        <groupId>com.google.cloud.tools</groupId>
+        <groupId>com.google.appengine</groupId>
         <artifactId>appengine-maven-plugin</artifactId>
-        <version>2.2.0</version>
-        <configuration>
-          <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
```

Comment:
Hmm, you know more about me when it comes to the inner workings of this portfolio, but just to be sure, is there a reason why you deleted this and didn't specify your project ID?